main.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"fmt"
+	"go1f/pkg/db"
+	"go1f/pkg/server"
+)
+
+func main() {
+	err := db.Init("scheduler.db")
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Можно так сделать, тогда `return` не нужен
```suggestion
		log.Fatalf("error in init db: %v", err)
```